charts/api7/templates/developer-portal-ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.developer_portal.ingress.enabled }}
+{{- $fullName := printf "%s-developer-portal" (include "api7ee3.fullname" .) -}}
+{{- $svcPort := .Values.developer_portal.port -}}
+{{- if and .Values.developer_portal.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.developer_portal.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.developer_portal.ingress.annotations "kubernetes.io/ingress.class" .Values.developer_portal.ingress.className}}
+  {{- end }}
+{{- end }}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "api7ee3.labels" . | nindent 4 }}
+  {{- with .Values.developer_portal.ingress.annotations }}
+  annotations:
```

Comment:
Need this backend tls annotaion like 
https://github.com/api7/api7-helm-chart/blob/f00f9670bae15cda887b54dea314b07bf0853e48/charts/api7/templates/dashboard-ingress.yaml#L24-L27
portal listen as tls endpoint by default.

Reply:
Currently HTTPS behavior on portal is controlled by an environment variable called `DISABLE_HTTPS`, rather than any configuration item. 🤔

Reply:
Are we going to write it as a configuration item?

Reply:
@nic-6443 I have added a TLS listen flag for enable or disable HTTPS listen of developer portal.